### DISCUSSION TITLE
terminal is not showing up in my codesandbox

### DISCUSSION THREAD

**Discussion by Sameedkhan25** [codesandbox/codesandbox-client#4253] [General]
i have created reactjs sandbox from the templates and terminal is not showing along with console? is this feature being removed or whats the issue here ? is there any settings to enable it?
![codesandbox issue](https://user-images.githubusercontent.com/22944289/83341579-2850d380-a2fe-11ea-83a7-cf37c6761e33.PNG)

**Comment by garethx** [comment]
Your sandbox is a client sandbox, which doesn't have support for a terminal. Since everything is bundled in the browser, no server exists on which a terminal can run. If you need to use a terminal, please create a container sandbox (fork the Node template from https://codesandbox.io/s/).

There's more info about the difference between container and client sandboxes here: https://codesandbox.io/docs/environment
